components/eamxx/docs/refs.bib

```diff
@@ -0,0 +1,125 @@
+VOLUME = {15},
+YEAR = {2022},
+NUMBER = {16},
+PAGES = {6285--6310},
+URL = {https://gmd.copernicus.org/articles/15/6285/2022/},
+DOI = {10.5194/gmd-15-6285-2022}
+}
+
+@article{Caldwell_et21,
+author = {Caldwell, P. M. and Terai, C. R. and Hillman, B. and Keen, N. D. and Bogenschutz, P. and Lin, W. and Beydoun, H. and Taylor, M. and Bertagna, L. and Bradley, A. M. and Clevenger, T. C. and Donahue, A. S. and Eldred, C. and Foucar, J. and Golaz, J.-C. and Guba, O. and Jacob, R. and Johnson, J. and Krishna, J. and Liu, W. and Pressel, K. and Salinger, A. G. and Singh, B. and Steyer, A. and Ullrich, P. and Wu, D. and Yuan, X. and Shpund, J. and Ma, H.-Y. and Zender, C. S.},
+title = {Convection-Permitting Simulations With the E3SM Global Atmosphere Model},
+journal = {Journal of Advances in Modeling Earth Systems},
+volume = {13},
+number = {11},
+pages = {e2021MS002544},
+keywords = {cloud resolving model, storm resolving model, general circulation model, convection permitting model, E3SM},
+doi = {https://doi.org/10.1029/2021MS002544},
+url = {https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2021MS002544},
+eprint = {https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1029/2021MS002544},
+note = {e2021MS002544 2021MS002544},
+year = {2021}
+}
+
+@article{Fiedler_Panofsky72,
+author = {Fiedler, F. and Panofsky, H. A.},
+title = {The geostrophic drag coefficient and the ‘effective’ roughness length},
+journal = {Quarterly Journal of the Royal Meteorological Society},
+volume = {98},
+number = {415},
+pages = {213-220},
+doi = {https://doi.org/10.1002/qj.49709841519},
+url = {https://rmets.onlinelibrary.wiley.com/doi/abs/10.1002/qj.49709841519},
+eprint = {https://rmets.onlinelibrary.wiley.com/doi/pdf/10.1002/qj.49709841519},
+abstract = {Abstract An ‘effective’ roughness length is defined for use over heterogeneous terrain as the roughness length which homogeneous terrain would have to give the correct surface stress over a given area. A method is suggested to compute geostrophic drag coefficient, wind-contour angle and surface heat flux, given this roughness length, latitude, geostrophic wind speed and insolation or ground-air temperature differences.},
+year = {1972}
+}
+
+
+@article{Hannah_et21,
+author = {Hannah, Walter M. and Bradley, Andrew M. and Guba, Oksana and Tang, Qi and Golaz, Jean-Christophe and Wolfe, Jon},
+title = {Separating Physics and Dynamics Grids for Improved Computational Efficiency in Spectral Element Earth System Models},
+journal = {Journal of Advances in Modeling Earth Systems},
+volume = {13},
+number = {7},
+pages = {e2020MS002419},
+keywords = {computational efficiency, E3SM, effective resolution, grid remap methods},
+doi = {https://doi.org/10.1029/2020MS002419},
+url = {https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2020MS002419},
+eprint = {https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1029/2020MS002419},
+note = {e2020MS002419 2020MS002419},
+year = {2021}
+}
+
+@article {Morrison_Milbrandt15,
```

Comment:
Do we care about consistent formatting here?
Or are we ok with it turning into a "dumping ground" of bibtex info?
(personally I'm ok relaxing my standards here)

Reply:
What consistency rules would you suggest? I think it doesn't really matter how you format bibtex entries because latex rules will make everything look the same once compiled. Or do you mean conventions for article labels?

Reply:
I was referring to all of that. The indentation is what made me think of it, but then I got to thinking whether should have a note about the format and provide a loose "template".

But I also think that's kinda dumb because it's likely people won't conform, so let's forget that I suggested this.